### ISSUE TITLE
v3 slide and empty page problem

### ISSUE THREAD

**Issue by webyonet** [nanxiaobei/react-slide-routes#30]
Functions that v2 normally do work incorrectly in v3

`You can watch the smooth running version in v2 in the video below.`

https://user-images.githubusercontent.com/973302/205830726-4577079f-588c-4508-8c3a-ed8be51fd5f4.mov

<br/>
<br/>

`You can watch the errors that occurred when I upgraded to v3 without making any changes, in the video below.`

https://user-images.githubusercontent.com/973302/205829969-2e9b4220-a05f-4e47-86ec-1451386e44ce.mp4

```javascript
export default function Router(): FunctionComponent {
    return (
        <SlideRoutes duration={TimeSpan.milliSecond.threeHundredFifty} destroy={true} timing="ease-in-out" animation="slide">
            <Route index={true} path={NavigationPath.dashboard} element={<Dashboard/>}/>
            <Route path={NavigationPath.conceptSplashPage} element={<ConceptSplash/>}/>
            <Route path={NavigationPath.conceptMainPage} element={<ConceptMainPage/>}/>
        </SlideRoutes>
    );
}
```

**Comment by nanxiaobei** [comment]
Can you provide a minimal reproducible demo of your project?

**Comment by webyonet** [comment]
@nanxiaobei sample demo project here
[https://github.com/webyonet/problem-demo](url)

the problem is exactly if you do a `redux` related action on page changes, v3 works with problems
I change theme with `redux` on page transitions and v3 starts working problematic, but there is no such problem in v2 and it works very well

**Comment by nanxiaobei** [comment]
Fixed in `3.0.3`, please try again.

**Comment by webyonet** [comment]
@nanxiaobei thanks problem fixed
